pyradiosky/skymodel.py

```diff
@@ -2751,23 +3104,23 @@ def write_healpix_hdf5(filename, hpmap, indices, freqs, nside=None, history=None
     """
     Write a set of HEALPix maps to an HDF5 file.
 
-    Deprecated. Use `SkyModel.write_healpix_hdf5` instead.
+    Deprecated. Use `SkyModel.write_skyh5` instead.
```

Comment:
also here.

Reply:
This one has a deprecation warning if it is called.